src/commands/retrieve/metadata.ts

```
@@ -138,8 +148,38 @@ export default class RetrieveMetadata extends SfCommand<RetrieveMetadataResult>
     const fileResponses = result?.getFileResponses() || [];
 
     if (!flags.json) {
-      displaySuccesses(result);
+      if (result.response.status === 'Succeeded') {
```

Errors were being eaten, so checking status and throw when not a success.